swift/Workflow/Sources/SubtreeManager.swift

```diff
@@ -338,7 +338,7 @@ extension WorkflowNode.SubtreeManager {
         }
 
         func handle(event: Output) {
-            if #available(iOS 10.0, OSX 10.12, *) {
```

Comment:
We don't support macOS, so I removed this extra availability check while auditing for the new iOS requirement.

Reply:
In theory we _could_ support macOS in `Workflow` itself. But probably best to keep that a separate concerted effort (with tests to prevent regressions as long as no one is using it that way).